test/e2e/scheduling/priorities.go

```diff
@@ -114,6 +114,7 @@ var _ = framework.KubeDescribe("SchedulerPriorities [Serial]", func() {
 
 		By("Create an RC with the same number of replicas as the schedualble nodes. One pod should be scheduled on each node.")
 		rc := createRC(ns, "scheduler-priority-selector-spreading", int32(len(nodeList.Items)), map[string]string{"name": "scheduler-priority-selector-spreading"}, f, podRequestedResource)
+		framework.WaitForRCToStabilize(cs, ns, rc.Name, framework.PodReadyBeforeTimeout)
```

Comment:
Suggest to also add it after https://github.com/kubernetes/kubernetes/pull/44309/files#diff-7c66db3bee38c600edec0d4f82136f3cR362